src/test/java/org/openlmis/stockmanagement/web/stockcardsummariesv2/StockCardSummariesV2DtoBuilderTest.java

```diff
@@ -232,94 +224,66 @@ public void shouldSortStockCardSummaries() throws Exception {
     assertEquals(result.get(1), summary2);
   }
 
-  @Test
-  public void shouldBuildStockCardSummariesWithDateBeforeThereWereCards() throws Exception {
-    List<StockCard> stockCards = asList(stockCard, stockCard1);
-
-    LocalDate asOfDate = getBaseDate().minusDays(10);
-
-    List<StockCardSummaryV2Dto> result = builder.build(Collections.singletonList(orderable1),
-        stockCards, fulfillMap, asOfDate, true);
-
-    StockCardSummaryV2Dto summary1 = new StockCardSummaryV2Dto(
-        new ObjectReferenceDtoDataBuilder()
-            .withPath(ORDERABLES)
-            .withId(orderable1.getId())
-            .build(),
-        asSet(
-            new CanFulfillForMeEntryDtoDataBuilder()
-                .buildWithStockCardAndOrderable(stockCard1, orderable3, asOfDate),
-            new CanFulfillForMeEntryDtoDataBuilder()
-                .buildWithStockCardAndOrderable(stockCard, orderable1, asOfDate))
-    );
-
-    assertEquals(1, result.size());
-    assertThat(result, hasItems(summary1));
-  }
-
-  @Test
-  public void shouldBuildStockCardSummariesForCurrentDate() throws Exception {
-    List<StockCard> stockCards = asList(stockCard, stockCard1);
-
-    LocalDate asOfDate = LocalDate.now();
-
-    List<StockCardSummaryV2Dto> result = builder.build(Collections.singletonList(orderable1),
-        stockCards, fulfillMap, null, false);
-
-    StockCardSummaryV2Dto summary1 = new StockCardSummaryV2Dto(
-        new ObjectReferenceDtoDataBuilder()
-            .withPath(ORDERABLES)
-            .withId(orderable1.getId())
-            .build(),
-        asSet(
-            new CanFulfillForMeEntryDtoDataBuilder()
-                .buildWithStockCardAndOrderable(stockCard1, orderable3, asOfDate),
-            new CanFulfillForMeEntryDtoDataBuilder()
-                .buildWithStockCardAndOrderable(stockCard, orderable1, asOfDate))
-    );
-
-    assertEquals(1, result.size());
-    assertThat(result, hasItems(summary1));
-  }
+  // We comment this test for now becouse filtering via date is done in Summary service  
```

Comment:
When should this be uncommented?

Reply:
Actually this should be removed as this filtering is not needed anymore as we are getting SoH as of date while getting stock cards with value from SoH fact table